src/Orchard.Web/Modules/Orchard.MediaLibrary.WebSearch/Providers/IWebSearchProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using Orchard.MediaLibrary.WebSearch.Models;
+using Orchard.MediaLibrary.WebSearch.ViewModels;
+
+namespace Orchard.MediaLibrary.WebSearch.Providers {
+    public interface IWebSearchProvider : IDependency {
+        string Name { get; }
+        IWebSearchSettings Settings { get; }
+        bool IsValid { get; }
+        string ApiKey { get; }
```

Comment:
This is already exposed through the Settings property, but a shortcut can be added as an extension method.